src/concrete/ml/sklearn/linear_model.py

```diff
@@ -254,27 +254,6 @@ def __init__(
                     f"({fit_encrypted=}). Got {parameters_range=}"
                 )
 
-    def post_processing(self, y_preds: numpy.ndarray) -> numpy.ndarray:
```

Comment:
we don't need it as this is essentially a (bad) copy paste of the linea models' `post_processing` method, which is inherited and used anyway